dataworkspace/dataworkspace/templates/datasets/index.html

```diff
@@ -106,6 +107,23 @@ <h2 class="govuk-heading-m">
         {% endif %}
       </ul>
     </nav>
+    {% endif %}
+
+    <div {% if datasets.paginator.count %}class="govuk-!-margin-top-9"{% endif %}>
+      <p class="govuk-body">
+        {% if datasets.paginator.count %}
+          If you haven't found what you're looking for, please:
+        {% else %}
+          There are no results for your search, please:
+        {% endif %}
+      </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>check the spelling of your keywords</li>
+        <li>use more general keywords</li>
+        <li>select or deselect different filters</li>
+        <li>request data via our <a href="{% url 'support' %}?tag=data-request">feedback and support form</a></li>
```

Comment:
We call it "support and feedback" across the rest of the site, so probably good to keep that order the same unless there's a reason to stray.
```suggestion
        <li>request data via our <a href="{% url 'support' %}?tag=data-request">support and feedback form</a></li>
```